eZ/Publish/Core/MVC/Symfony/View/RelationView.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\View;
+
+/**
+ * A view that contains relation data.
+ */
+interface RelationView
+{
+    /**
+     * Returns the Relations for the current entity being viewed.
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Relation[]
```

Comment:
@bdunogier Note: I assumed you would prefer full object, but for tag needs I just to keep id's around, and with that we could optimize the loading of relational data to only be one spi calls instead of 1+n _(where n is number of relations)_. The design of this does not make a differance when cached, but it does make a differance in dev _(either on cold cache or with cache disabled)_.